docs/docs/labelling-issues.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Labelling issues
+---
+
+## What are issue labels?
+
+Issue labels are a tool in GitHub that that is used to group issues into one or more categories.
+
+[Check out Gatsby's labels (and their descriptions)](https://github.com/gatsbyjs/gatsby/issues/labels)
+
+
+## Why label issues?
+
+Gatsby is a very active project with many new issues opened each day. Labelling issues helps by identifying:
+
+- good issues for new contributors to work on
+- reported and confirmed bugs
+- new features
+- duplicate issues
+- issues that are stalled or blocked
+
+## Who can label issues?
+
+Anyone who's a member of the [Gatsby Maintainers team](https://github.com/orgs/gatsbyjs/teams/maintainers) can label issues.
+
+You can get an invite to the team by having a Pull Request merged into the Gatsby project. Check out the list of [`help wanted`](https://github.com/gatsbyjs/gatsby/labels/%F0%9F%93%8D%20status%3A%20help%20wanted) issues and the [How To Contribute Guide](/docs/how-to-contribute/) to get started.
```

Comment:
Let's add a note:

**NOTE:** If you’ve already had a pull request merged and you have _not_ been invited to the maintainers team, please go to [the dashboard](https://store.gatsbyjs.org/account/dashboard) and request a discount code. You should get an invite to the team — _and you get free Gatsby swag!_ If that doesn’t work, please email team@gatsbyjs.com and we'll get you invited.